packages/venia-ui/lib/components/PageLoadingIndicator/indicator.css

```diff
@@ -17,6 +17,12 @@
     }
 }
 
+@media (min-width: 1024px) {
+    .root {
+        grid-area: primary;
```

Comment:
I would suggest using grid columns and rows instead of grid areas.

Reply:
Fixed